plugins/source/aws/resources/services/fsx/file_caches_fetch.go

```diff
@@ -0,0 +1,46 @@
+package fsx
+
+import (
+	"context"
+	"errors"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/fsx"
+	"github.com/aws/aws-sdk-go-v2/service/fsx/types"
+	"github.com/aws/smithy-go"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/rs/zerolog/log"
+)
+
+func fetchFsxFileCaches(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	cl := meta.(*client.Client)
+	svc := cl.Services().Fsx
+	input := fsx.DescribeFileCachesInput{MaxResults: aws.Int32(1000)}
+	paginator := fsx.NewDescribeFileCachesPaginator(svc, &input)
+	for paginator.HasMorePages() {
+		result, err := paginator.NextPage(ctx)
+		if err != nil {
+			var ae smithy.APIError
+			if errors.As(err, &ae) && ae.ErrorCode() == "BadRequest" && ae.ErrorMessage() == "The requested feature is not enabled for this AWS account." {
+				log.Warn().Err(err).Msg("skipping resource")
```

Comment:
warn the user? not warn the user? ⚖️

Reply:
Is this feature enabled by default or disabled? If it's enabled by default we should throw an error, otherwise we can print a warning (I think)